docs/Get started/FAQ.md

```diff
@@ -335,6 +335,8 @@ There are many challenges to generating .msi packages on any OS but Windows. Err
 Packs are a function of osquery that provide a portable format to import /export queries in and out of platforms like Fleet. These osquery packs still exist, but have been removed from the Fleet UI. Access via API is still available for backwards compatibility.
 Within Fleet we've introduced the concept of teams in Fleet premium to target specific groups of hosts, but you can also still use scheduled queries in Fleet free (works like packs) to target all your hosts.
 
+The `fleetctl upgrade-packs` command can be used to convert existing packs to queries.
```

Comment:
Seeing this in context, I'm realizing we probably need to update the "Where did Packs go?" answer to be more aligned with the "Why this way?" [section about packs](https://fleetdm.com/handbook/company/why-this-way#why-does-fleet-support-query-packs). That said, this change totally fits with the _existing_ content, so I don't want to let that hold up this PR— just saying this as an FYI.

cc @zhumo not sure how we track inconsistencies in the docs right now so just filed it as a [bug](https://github.com/fleetdm/fleet/issues/13191); happy to take that one on after this PR is merged.

Reply:
I think that's a good approach Rachael, please go ahead.